index.html

```diff
@@ -166,4 +166,30 @@ <h1>About <br> Myself</h1>
     </div>
   </section>
 </body>
-</html>
\ No newline at end of file
+
+<footer class="formsection">
+  <h1>Contact me</h1>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+  
+  <form>
```

Comment:
- In order to properly submit your form and receive user input, you are required to integrate your form with [formspree service](https://formspree.io/html).
- Your form should have an `action` value similar to `action = "https://formspree.io/f/{form_id}"`
- Your form should have a `method` value equal to `post`